plugins/importexport/datacite/locale/pt_PT/common.xml

```diff
@@ -2,55 +2,67 @@
 <!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
 
 <!--
-  * @file plugins/importexport/medra/locale/pt_PT/common.xml
+  * plugins/importexport/datacite/locale/pt_PT/common.xml
   *
   * Copyright (c) 2013-2016 Simon Fraser University Library
   * Copyright (c) 2003-2016 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
-  * Shared localization strings
+  * Shared localization strings.
```

Comment:
I don't mind adding this to the comment, but I wonder why you introduced this and maybe we will overwrite this in the future when updating headers.